### DISCUSSION TITLE
Storage

### DISCUSSION THREAD

**Discussion by shyamganesh-dev** [AppFlowy-IO/AppFlowy#3668] [General help wanted]
Where our data is stored? cloud or local?
is it good for small teams?

**Comment by annieappflowy** [comment]
It's 100% local by default. You can go to Settings -> Files to check where the data is stored.
We're running a private beta program for AppFlowy Cloud. You can sign up for it [here](https://survey.appflowy.io/private-beta)
With AppFlowy Cloud, you can opt for end-to-end encryption, ensuring utmost privacy and security of your data. In addition, you will have the flexibility to remain local and sync as required.
Team collaboration is in the development. The estimated delivery time is December.

**Reply by Eqxlt** [reply]
Is E2EE still an option for the cloud beta? I thought I opted for it, but I can't tell as I don't see it as an option anymore.

**Comment by shyamganesh-dev** [comment]
im a  brand/graphic designer.. i like the appflow so i would like to help
annie..
